### PR TITLE
Gateway route prototype initial stage implementation

### PR DESCRIPTION
**Pull request by sumedhsalvi**
I have implemented a gateway route prototype for the initial stage

### CODE REVIEW

File: weather-radar-gateway/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const router = express.Router()
+
+router.all('/:apiName', (req, res) => {
```
Comment (sanket1211996) [inline weather-radar-gateway/routes/index.js:5]: Add comment explaining working of the service.

**Review by sanket1211996 — COMMENTED** [review]
Good for basic implementation.

File: weather-radar-gateway/routes/index.js (hunk 1 above)
Comment (sumedhsalvi) [reply weather-radar-gateway/routes/index.js:5]: I have added comments. Please review again.

Comment (sanket1211996) [reply weather-radar-gateway/routes/index.js:5]: Its looks good. Merging

**Pull request merged**